client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.html

```diff
@@ -223,7 +223,7 @@ <h3 class="oui-tile__title">
                                         </div>
                                         <button type="button"
                                                 class="btn btn-default ml-auto align-self-center"
-                                                data-ng-if="options[optionName].state === 'disabled' && allDisabled() && !anyEnabling() && nsxAndVropsCompliantPcc()"
+                                                data-ng-if="target !== 'US' && options[optionName].state === 'disabled' && allDisabled() && !anyEnabling() && nsxAndVropsCompliantPcc()"
```

Comment:
Does `featureAvailability` service seems to be a better option for that kind of case?